docs/references/general/grid.md

```diff
@@ -0,0 +1,101 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data produced from FITACF files.
+
+## Naming Conventions
+
+### Single radar GRID files
+
+Currently the common naming convention for a single radar GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+Some radars provide separate data files for each channel. In this case, the channel is specified after the 3-letter station ID:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) or by combining individual GRID files with `combine_grid`, then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined "GRD" files for each hemisphere
```

Comment:
```suggestion
### Combined GRID files for each hemisphere
```
Do we want to fully drop this `GRD` naming convention?

Reply:
@egthomas Ahh yes, we did discuss that a few weeks ago. It's getting late here now so I can do this tomorrow.